tasks/nc_installation.yml

```diff
@@ -175,3 +175,14 @@
     - config
     - themes
     - updater
+
+- name: "[NC apps] - Disable Nextcloud apps"
```

Comment:
@staticdev What do you think about moving this whole step to [nc_apps.yml](https://github.com/aalaesar/install_nextcloud/blob/139ec82d1d7a2df7f4c9a5d93b7b57c7d32cdf9c/tasks/nc_apps.yml) to have all app management within one file?

Reply:
Hello.
@staticdev @wiktor2200
I think it's good as is for one last release for 1.x
It'll be reworked on v2 with a module to manage apps.
Probably one of the first modules I'll work on.

Reply:
Still need a Readme example and some CI tests 😀

Reply:
@aalaesar maybe we can drop this one an refactor when we have the occ module from https://github.com/aalaesar/install_nextcloud/pull/198

Reply:
@staticdev hm. We are just missing some test and doc. Let me finish this today so we'll merge it Before 2.0. Idealy a manage app module would bring more  value